usermods/TetrisAI_v2/usermod_v2_tetrisai.h

```diff
@@ -151,48 +153,48 @@ uint16_t mode_2DTetrisAI()
     tetrisai_data->intelligence = SEGMENT.custom1;
     double dui = 0.2 - (0.2 * (tetrisai_data->intelligence / 255.0));
 
-    tetrisai_data->tetris.ai.aHeight = -0.510066 + dui;
-    tetrisai_data->tetris.ai.fullLines = 0.760666 - dui;
-    tetrisai_data->tetris.ai.holes = -0.35663 + dui;
-    tetrisai_data->tetris.ai.bumpiness = -0.184483 + dui;
+    tetris.ai.aHeight = -0.510066 + dui;
+    tetris.ai.fullLines = 0.760666 - dui;
+    tetris.ai.holes = -0.35663 + dui;
+    tetris.ai.bumpiness = -0.184483 + dui;
   }
 
-  if (tetrisai_data->tetris.state == TetrisAIGame::ANIMATE_MOVE)
+  if (tetris.state == TetrisAIGame::ANIMATE_MOVE)
   {
-    if (millis() - tetrisai_data->lastTime > msDelayMove)
+    if (millis() - lastTime > msDelayMove)
     {
-      drawGrid(&tetrisai_data->tetris, tetrisai_data);
-      tetrisai_data->lastTime = millis();
-      tetrisai_data->tetris.poll();
+      drawGrid(&tetris, tetrisai_data);
+      lastTime = millis();
+      tetris.poll();
     }
   }
-  else if (tetrisai_data->tetris.state == TetrisAIGame::ANIMATE_GAME_OVER)
+  else if (tetris.state == TetrisAIGame::ANIMATE_GAME_OVER)
   {
-    if (millis() - tetrisai_data->lastTime > msDelayGameOver)
+    if (millis() - lastTime > msDelayGameOver)
```

Comment:
Please use `strip.now` instead of `millis()`

Reply:
OK it should be very easy to change that.